src/app/store.ts

```diff
@@ -1,6 +1,15 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
```

Comment:
The import statement `combineSlices` is incorrect. You should use `combineReducers` from `@reduxjs/toolkit` to combine the slices into a root reducer.